components/servo/servo_delegate.rs

```diff
@@ -38,6 +38,11 @@ pub trait ServoDelegate {
 
     /// Request to display a notification.
     fn show_notification(&self, _notification: Notification) {}
+
+    /// A console message was logged by content not associated with a specific [`WebView`].
```

Comment:
Please link to the console API here: https://developer.mozilla.org/en-US/docs/Web/API/Console_API